### PR TITLE
PLATUI-321: Added sbt-auto-build plugin

### PR DESCRIPTION
**Pull request by JoPintoPaul**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -8,3 +8,5 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.22.0")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.5.2")
 
 addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.6.4")
+
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.22.0")
```
Comment (funkmaster20th) [inline project/plugins.sbt:12]: is this not a duplicate?

**Review by TimothyFothergill — DISMISSED** [review]
lgtm

File: project/plugins.sbt (hunk 1 above)
Comment (funkmaster20th) [reply project/plugins.sbt]: resolved :)

**Review by vivrichards600 — APPROVED** [review]

**Review by funkmaster20th — APPROVED** [review]

**Comment by oscarduignan** [comment]
test this please

**Pull request merged**